kit/azure/bootstrap/resources.docs-spn.tf

```diff
@@ -0,0 +1,42 @@
+# todo: this could be nicer packaged as it's own submodule, so that it doesn't need all those count = ... checks
+
+resource "azurerm_user_assigned_identity" "docs" {
```

Comment:
f: I'm afraid we can't or should not backport this 1:1 from livkid bank as it has some hardcoded assumptions in there that need proper modularization.

Reply:
I took it complete out of the hub yet